flat_postprocess/oui_postprocess.py

```diff
@@ -24,6 +24,6 @@
         oui_mappings.write("# pylint: disable=too-many-lines\n")
         oui_mappings.write("import typing\n\n")
         oui_mappings.write("OUI_MAPPINGS: typing.Dict[str, str] = {\n")
-        for mac, company in OUI_MAPPINGS.items():
+        for mac, company in sorted(OUI_MAPPINGS.items()):
```

Comment:
@jeffkala So I wasn't doing any ordering. I guess I assumed what I received from the web would be consistent in terms of placement. This should order the `OUI_MAPPINGS` dict before writing to the file in the repo.  I'm guessing there will be a lot of changes on the initial PR but after that it should just notice actual additions and removals from the data. Thoughts?

Reply:
this works for me. Doesn't have to be fancy, but this will help for future PRs.